src/main/resources/templates/db_scripts/schema.json

```diff
@@ -101,6 +101,12 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": false
+        },
+        {
+          "fieldName": "status.name",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": true
```

Comment:
Has it been confirmed that this setting will align with a general CQL query for this property. There is a mixture of definitions for the other indexes.

Reply:
I have copied format from other `status.name` indexes.